lib/Messenger/Handler/SearchBackendHandler.php

```diff
@@ -35,6 +38,29 @@ public function __invoke(SearchBackendMessage $message, Acknowledger $ack = null
         return $this->handle($message, $ack);
     }
 
+    private function processElement(Element\ElementInterface $element, bool $updateChildren) {
+
+        $searchEntry = Data::getForElement($element);
+        if ($searchEntry->getId() instanceof Data\Id) {
+            $searchEntry->setDataFromElement($element);
+        } else {
+            $searchEntry = new Data($element);
+        }
+        $searchEntry->save();
+
+        if ($updateChildren) {
+            if ($element instanceof Asset || $element instanceof AbstractObject || $element instanceof Document) {
```

Comment:
This is redundant since we have already `Element\ElementInterface $element` in the arguments 😉